src/utils/assets.js

```diff
@@ -75,3 +75,12 @@ export function normalizeAssetUrl(url) {
   }
   return pathJoin([baseUrl, url]);
 }
+
+/**
+ * Creates absolute URLs from paths
+ * @param {String} path
+ * @return {String}
+ */
+export function absoluteURL(path) {
```

Comment:
Would it be possible to avoid the need for this function if the images were included in the assets folder and we used the normal static asset handling that vue-cli [provides](https://cli.vuejs.org/guide/html-and-static-assets.html)?

Either way, we will likely want to ensure that we utilize the `normalizeAssetUrl` function so that these images still work as expected when a custom base path is used for a developer's website (we should probably manually test this branch with a GitHub Pages workflow before merging to double check that there are no problems there).

Reply:
I moved the images to the HTML as you suggested, thanks for that!

The reason why I was using this `absoluteURL` function is because we can only use absolute urls for metadata (instead of relative ones as in the rest of the app) but it's true that `normalizeAssetUrl` should render the proper absolute path when being built, so it should be fine.